packages/mdc-radio/index.js

```diff
@@ -84,15 +102,24 @@ export class MDCRadio extends MDCComponent {
     return new MDCRipple(this.root_, foundation);
   }
 
+  /**
+   * Returns the state of the native control element, or undefined if the native control element is not present.
+   * @return {SelectionControlState|undefined}
+   * @private
+   */
   get nativeControl_() {
-    return this.root_.querySelector(MDCRadioFoundation.strings.NATIVE_CONTROL_SELECTOR);
+    const {NATIVE_CONTROL_SELECTOR} = MDCRadioFoundation.strings;
+    const el = /** @type {SelectionControlState|undefined} */ (
```

Comment:
`{SelectionControlState|undefined}` -> `{?SelectionControlState}`

`querySelector()` returns `null`, not `undefined`.